server/middleware/search.js

```diff
@@ -1,15 +1,14 @@
 /* eslint-disable no-mixed-spaces-and-tabs */
 const YouTube = require('youtube-node');
-
-const youTube = new YouTube();
-youTube.setKey(process.env.KEY_YOUTUBE);
+const SC = require('node-soundcloud');
```

Comment:
why no es6 import syntax?

Reply:
There isn't a reason really, the server side was already using requires so I kept the trend going.